lib/plugin/gitlab/GitLab.js

```diff
@@ -17,10 +17,15 @@ class GitLab extends Release {
     super(...args);
     this.registerPrompts(prompts);
     this.assets = [];
-    const { certificateAuthorityFile } = this.options;
-    this.certificateAuthorityOption = certificateAuthorityFile
-      ? { https: { certificateAuthority: fs.readFileSync(certificateAuthorityFile) } }
-      : {};
+    const { certificateAuthorityFile, secure } = this.options;
+
+    const httpsOptions = {
+      certificateAuthority: certificateAuthorityFile ? fs.readFileSync(certificateAuthorityFile) : undefined,
+      rejectUnauthorized: typeof secure === 'boolean' ? secure : undefined
+    };
+    const https = _.merge({}, httpsOptions);
```

Comment:
Looks like we don't need this intermediate step?

Reply:
That's correct, we don't really need it. My idea here was to use lodash merge because it remove keys with undefined values, so we wouldn't end up setting `https` to an empty object, which was what the code was doing before. But I can remove this line.

Regarding testing, I tried to find a test using CA certificates, but couldn't find one, so I skipped it. Or do you mean just a unit test to check if the options passed to the Gitlab plugin are correct?

Reply:
> That's correct, we don't really need it. My idea here was to use lodash merge because it remove keys with undefined values, so we wouldn't end up setting `https` to an empty object, which was what the code was doing before. But I can remove this line.

Thanks, `merge` is not descriptive towards that end, but let's just keep it then.

> Regarding testing, I tried to find a test using CA certificates, but couldn't find one, so I skipped it. Or do you mean just a unit test to check if the options passed to the Gitlab plugin are correct?

Just the latter!